app/src/components/market/common/currency_selector/index.tsx

```diff
@@ -11,6 +11,10 @@ const Wrapper = styled.div`
   display: flex;
   flex-wrap: wrap;
   width: 100%;
+
+  .token_item_wrapper {
+    color: ${props => props.theme.colors.textColorDark};
+  }
```

Comment:
We should place this in the `token_item` file where it's being used for easier maintainability